agr_java_core/src/main/java/org/alliancegenome/api/entity/GeneDiseaseAnnotationDocument.java

```diff
@@ -18,4 +20,24 @@ public class GeneDiseaseAnnotationDocument extends DiseaseAnnotationDocument {
 	public GeneDiseaseAnnotationDocument() {
 		setCategory("gene_disease_annotation");
 	}
+
+	// 1 true
+	// 0 false
+	@JsonIgnore
+	private boolean isViaOrthologyAnnotation;
+
+	@JsonView({View.DiseaseAnnotationAll.class})
+	public int getViaOrthologyOrder() {
+		return isViaOrthologyAnnotation ? 1 : 0;
+	}
+
+	@JsonView({View.DiseaseAnnotationAll.class})
+	public void setViaOrthologyOrder(int order) {
+		isViaOrthologyAnnotation = order == 1;
+	}
+
+	@Override
```

Comment:
Can we remove the toString this should be done via `@ToString` at the class level.

Reply:
Removed.